skyagi/src/skyagi/skyagi.py

```diff
@@ -34,10 +36,14 @@ def user_robot_conversation(agent_to_interview: GenerativeAgent, ctx: Context):
             )
         if "GOODBYE" in response:
             ctx.print(
-                f"{agent_to_interview.name} said Goodbye and ended the conversation"
+                message=f"{agent_to_interview.name} said Goodbye and ended the conversation",
```

Comment:
we might not need to put the `{agent_to_interview.name} said` in the message. Are you gonna parse the message in BE?